docker/Makefile

```diff
@@ -41,11 +44,21 @@ ifdef no_proxy
 endif
 ifdef DO_PROXY
    DOCKER_COMPOSE_FILES += sawtooth-pdo.proxy.yaml
+   DOCKER_COMPOSE_FILES_CCF += ccf-pdo.proxy.yaml
 endif
 ifdef PDO_DEBUG_BUILD
    DOCKER_COMPOSE_FILES += sawtooth-pdo.debugging.yaml
 endif
+ifdef PDO_LEDGER_TYPE # not sure how to pull this check as part of the test-ccf target, so that we do not need the extra env PDO_LEDGER_TYPE
+ifeq ($(PDO_LEDGER_TYPE),ccf)
```

Comment:
Thank you, the second option is defintely preferred once we have the equivalent of all compose files for ccf as well. Currently, I only have the CCF SIM mode compose test. (not the HW  nor the local-code nor the debugging files). We cna move to the second option after we add  support for all of these.

And on the first point, yes, I would have liked to do this, but I am not sure how to do this.

Reply:
the debug i would anyway support (it's trivial) and for sgx i wouldn't worry. It would just fail iff somebody would try to run with PDO_LEDGER_TYPE=ccf & SGX_MODE=HW but that's fine; actually i think it would fail even right now and the error of missing compose-files might be more obvious why it fails than otherwise?